src/App.tsx

```diff
@@ -1,97 +1,81 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
+const itemsPerPageOptions = ['3', '5', '10', '20'];
+
 export const App: React.FC = () => {
+  const [itemsPer, setItemsPer] = useState(5);
```

Comment:
I would do: 
```suggestion
  const [itemsPer, setItemsPer] = useState(itemsPerPageOptions[1]);
```

This way you will only change it in one place.